docs/docs/06-transferring-props.ru-RU.md

```diff
@@ -0,0 +1,154 @@
+---
+id: transferring-props-ru-RU
+title: Передача свойств
+permalink: transferring-props-ru-RU.html
+prev: reusable-components-ru-RU.html
+next: forms.html
+---
+
+В React распространен шаблон, заключающийся в построении абстракции вокруг компонента. Внешний компонент выставляет единственное свойство для достижения чего-то, что, возможно, имеет сложную внутреннюю реализацию.
+
+Вы можете использовать [расширение атрибутов JSX](/react/docs/jsx-spread-ru-RU.html), чтобы объединить старые свойства с дополнительными значениями:
+
+```javascript
+<Component {...this.props} more="values" />
+```
+
+Если вы не пользуетесь JSX, вы можете пользоваться любыми вспомогательными методами, такими как `Object.assign` в ES6 или `_.extend` из Underscore:
+
+```javascript
+React.createElement(Component, Object.assign({}, this.props, { more: 'values' }));
+```
+
+Оставшаяся часть этой статьи объясняет лучшие практики. Здесь используется JSX и экспериментальный синтаксис ECMAScript.
+
+## Передача вручную
+
+Большую часть времени вам следует явно передавать свойства вниз по иерархии. Это гарантирует, что вы выставляете наружу только подмножество внутреннего API, в работоспособности которого вы уверены.
+
+```javascript
+function FancyCheckbox(props) {
+  var fancyClass = props.checked ? 'FancyChecked' : 'FancyUnchecked';
+  return (
+    <div className={fancyClass} onClick={props.onClick}>
+      {props.children}
+    </div>
+  );
+}
+ReactDOM.render(
+  <FancyCheckbox checked={true} onClick={console.log.bind(console)}>
+    Hello world!
+  </FancyCheckbox>,
+  document.getElementById('example')
+);
+```
+
+Но что насчет свойства `name`? Или свойства `title`? Или `onMouseOver`?
+
+## Передача с помощью `...` в JSX
+
+> ЗАМЕЧАНИЕ:
+>
+> Синтаксис `...` является частью предложения об оставшихся свойствах и расширении свойств объектов. Это предложение на пути к тому, чтобы стать стандартом. Обратитесь к главе [Оставшиеся свойства и расширение свойств ...](/react/docs/transferring-props-ru-RU.html#-----...), чтобы узнать об этом подробнее.
```

Comment:
FWIW, we're actually trying to do this ourselves (https://github.com/facebook/react/blob/master/docs/_plugins/header_links.rb). We can probably make it better!